libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -223,4 +224,13 @@ private static Integer getAge(String dobValue) {
             return null;
         }
     }
+
+    public static Vot getVcVot(SignedJWT vc) throws UnrecognisedVotException {
+        try {
+            String vot = vc.getJWTClaimsSet().getStringClaim(VOT_CLAIM_NAME);
+            return vot == null ? null : Vot.valueOf(vot);
```

Comment:
Do we ever expect a null vot?

Reply:
Yea, there are cases where the vet would be null, I might be wrong but I think it would be null in most cases except for inherited identities.

Reply:
Yeah, we do expect to get null's here. Rather do the check than throw an NPE and handle it.